packages/admin-panel/src/api/TupaiaApi.js

```diff
@@ -10,7 +10,7 @@ import { verifyResponseStatus, stringifyQuery } from '@tupaia/utils';
 
 import { logout } from '../authentication';
 
-const FETCH_TIMEOUT = 45 * 1000; // 45 seconds in milliseconds
+const FETCH_TIMEOUT = 120 * 1000; // 45 seconds in milliseconds
```

Comment:
This is because a survey import could actually take more than 45 seconds now that it doesn't 'fail fast'